### DISCUSSION TITLE
Artifact Update Problem

### DISCUSSION THREAD

**Discussion by dbimhoff** [renovatebot/renovate#10473] [Request Help]
Hello, I am implementing realm on our self hosted git lab and is configured to run on gitlab runners. When I run renovate it makes the PRs as expected, but for some reason I get an Artifact Update Problem on the react v17 and react-dom v17. 

Image of PRs being created: As you can see the top 2 have comments on them 
![image](https://user-images.githubusercontent.com/47460175/122295790-42386680-cebf-11eb-8e3a-c7c21d02472a.png)

React v17 comment:
![image](https://user-images.githubusercontent.com/47460175/122295974-77dd4f80-cebf-11eb-9ba8-531322df4acf.png)

React-Dom v17 comment:
![image](https://user-images.githubusercontent.com/47460175/122296015-83c91180-cebf-11eb-89fa-fc75a6bf9f40.png)

I am not sure what is causing this issue. Could you help me at all? I can provide the config.json or other stuff if needed. Thanks.

**Comment by viceice** [comment]
Please check the debug logs of renovate, it seems there is a npm version mismatch

**Reply by rarkins** [reply]
Is it npm v6 or npm v7 which is running? npm v7 has strict peer dependency resolution which can "intentionally" cause this type of error

**Reply by dbimhoff** [reply]
This is running npm v7. Also I'm a little new to npm v7, how can I go about fixing this?

**Reply by dbimhoff** [reply]
nvm I think I have fixed it with a .npmrc w/ --legacy-peer-deps
